web/src/locales/fr.json

```diff
@@ -29,6 +29,10 @@
     "areas": "Zones",
     "about": "À propos"
   },
+  "app-banner": {
+    "cta": "Téléchargez l'application",
```

Comment:
```suggestion
    "cta": "Téléchargez l'application",
```

This title is very long, but according to some random image on Google we might be able to get away with an alternative wording instead :)

![Screenshot 2024-09-11 at 08 27 42](https://github.com/user-attachments/assets/2e2eac24-fbe3-4e26-9e1c-d2ec732d5303)

![Screenshot 2024-09-11 at 08 27 36](https://github.com/user-attachments/assets/c12f01a3-9337-42c1-a27f-1db85d9cb309)

Maybe to be confirmed by an actual French speaking person :D

Reply:
🤣 Good catch, thanks! I ran it through chatgpt and that was the translation that I got back. Will def ask in the office today